tests/unit/gateways/clients/responses/hathor_wallet_service/test_hathor_wallet_service_token_response.py

```diff
@@ -0,0 +1,102 @@
+import pytest
+from dacite.exceptions import MissingValueError
+
+from gateways.clients.responses.hathor_wallet_service.hathor_wallet_service_token_response import (
+    HathorWalletServiceTokenListResponse,
+    HathorWalletServiceTokenResponse,
+    HathorWalletServiceTokenTransactionsReponse,
+)
+from tests.fixtures.hathor_wallet_service_fixtures import (
+    HATHOR_WALLET_SERVICE_GET_TOKEN,
+    HATHOR_WALLET_SERVICE_GET_TOKEN_LIST,
+    HATHOR_WALLET_SERVICE_GET_TOKEN_TRANSACTION_LIST,
+)
+
+
+class TestHathorWalletServiceTokenResponse:
```

Comment:
Same thing here. I don't know whether this test is useful since we are returning exactly the same thing as the wallet service.